pom.xml

```diff
@@ -208,6 +219,12 @@
 			<artifactId>mysql-connector-java</artifactId>
 			<version>5.1.14</version>
 		</dependency>
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<version>1.4.200</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Is there any Java6 compliant solution?

Reply:
http://h2-database.66688.n3.nabble.com/Java-6-compatibility-td4033435.html 
version 1.4.190 should be ok 
DATABASE_TO_LOWER is no longer needed because test db is generated by hibernate now so downgrade was possible.

11 tests left to fix and task will be ready

Reply:
Yeah, I did the samea adjustments, but I was not able to solve the problem on mysql DATE() function, see comments on https://github.com/informatici/openhospital-core/commit/eeee2b565aea59ae04e1ed85bcfe74f088a9a1e2